Common/Source/Dialogs/dlgProgress.cpp

```diff
@@ -134,3 +134,11 @@ void CreateProgressDialog(const TCHAR* text) {
         pWndProgress->SetProgressText(text);
     }
 }
+
+
+void ProgressDialogText(const TCHAR* text) {
```

Comment:
unused and useless...

Reply:
not true, needed to update text vie Timer event.

Reply:
if "ProgressDialogText" is used, say to me where ...